Sources/MapboxDirections/RouteRefreshSource.swift

```diff
@@ -31,6 +36,10 @@ extension RefreshedRoute: RouteRefreshSource {
     }
 }
 extension RefreshedRouteLeg: RouteLegRefreshSource {
+    public var refreshedIncidents: [Incident]? {
+        incidents
+    }
+    
     public var refreshedAttributes: RouteLeg.Attributes {
         attributes
     }
```

Comment:
NIT: it would be nice to keep the order of methods the same
```suggestion
    public var refreshedAttributes: RouteLeg.Attributes {
        attributes
    }

    public var refreshedIncidents: [Incident]? {
        incidents
    }
```